<!doctype html>
<html>
  <head>
    <title>LGSF Dashboard</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <link rel="shortcut icon" href="https://democracyclub.org.uk/static/images/logo_icon.png">
  </head>
  <body>
    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  


  <h2 id="2023-10-25-09-43">2023-10-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-25 09:43:38.285744</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-25 09:43:40.571800</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[09:43:38] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[09:43:39] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[09:43:40] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           503 Server Error: Service Unavailable for url:         handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-24-08-48">2023-10-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-24 08:48:38.372748</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-24 08:48:40.595981</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[08:48:38] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[08:48:39] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[08:48:40] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           503 Server Error: Service Unavailable for url:         handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-23-09-59">2023-10-23</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-23 09:59:22.239533</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-23 09:59:24.301783</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[09:59:22] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[09:59:23] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[09:59:24] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-22-10-41">2023-10-22</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-22 10:41:14.701651</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-22 10:41:16.800477</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:41:14] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[10:41:15] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:41:16] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-21-10-37">2023-10-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-21 10:37:24.732106</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-21 10:37:26.935629</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:37:24] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[10:37:25] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:37:26] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-20-08-41">2023-10-20</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-20 08:41:59.281714</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-20 08:42:01.514856</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[08:41:59] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[08:42:00] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[08:42:01] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-19-08-26">2023-10-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-19 08:26:43.915637</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-19 08:26:46.343094</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[08:26:43] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[08:26:44] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[08:26:45] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[08:26:46] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-18-09-08">2023-10-18</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-18 09:08:59.511206</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-18 09:09:01.661740</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[09:08:59] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[09:09:00] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[09:09:01] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-17-10-21">2023-10-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-17 10:21:54.480108</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-17 10:21:58.562802</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:21:54] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[10:21:55] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:21:56] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[10:21:58] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-16-09-18">2023-10-16</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-16 09:18:04.516336</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-16 09:18:06.754440</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[09:18:04] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[09:18:05] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[09:18:06] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-15-10-11">2023-10-15</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-15 10:11:51.157382</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-15 10:11:53.398206</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:11:51] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[10:11:52] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:11:53] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-14-10-25">2023-10-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-14 10:25:38.908231</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-14 10:25:41.299913</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 180, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 197, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:25:38] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[10:25:39] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:25:40] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[10:25:41] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-13-10-35">2023-10-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-13 10:35:03.268150</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-13 10:35:05.644618</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[10:35:03] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[10:35:04] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[10:35:05] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-13-09-40">2023-10-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-13 09:40:36.460253</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-13 09:40:40.095052</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[09:40:36] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[09:40:38] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[09:40:39] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[09:40:40] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-12-12-44">2023-10-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-12 12:44:04.927317</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-12 12:44:08.080543</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:44:04] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:44:05] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:44:06] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:44:07] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[12:44:08] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-11-12-30">2023-10-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-11 12:30:58.858846</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-11 12:31:01.106471</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:30:58] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:30:59] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:31:00] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
[12:31:01] Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-10-13-43">2023-10-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-10 13:43:32.327338</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-10 13:43:34.636815</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:43:32] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:43:33] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:43:34] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-09-12-04">2023-10-09</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-09 12:04:57.744906</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-09 12:04:59.820717</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:04:57] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:04:58] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:04:59] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-08-14-11">2023-10-08</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>1 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-08 14:11:39.990916</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-08 14:11:41.986967</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:11:39] Fetching Scraper for: HUN                              handlers.py:23
[14:11:40] Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:11:41] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-10-07-12-36">2023-10-07</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>1 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-10-07 12:36:18.748446</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-10-07 12:36:20.743742</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:36:18] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:36:19] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:36:20] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  


      </main>
      <footer class="ds-footer">
        <div class="ds-block-centered ds-text-centered ds-stack">
          <div class="ds-cluster-center">
            <ul>
              <li><a href="/lgsf-dashboard/api/failing.json">Failing JSON</a></li>
            </ul>
          </div>
          <div class="ds-copyright">
            <a href="https://democracyclub.org.uk/">
              <img src="https://DemocracyClub.github.io/design-system/images/logo_white_text.svg" alt="Democracy Club Home" />
            </a>
            <p>Copyright © 2021 Democracy Club</p>
            <p>Community Interest Company</p>
            <p>Company No: <a href="https://beta.companieshouse.gov.uk/company/09461226">09461226</a></p>
          </div>
        </div>
      </footer>
    </div>
  </body>
